src/web/index.html

```diff
@@ -1,250 +1,175 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="utf-8" />
+<head> 
+    <meta charset="utf-8">
     <title>HakuNeko</title>
-    <link rel="icon" href="./img/logo.png" />
-    <meta
-      name="viewport"
-      content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes"
-    />
-
+    <link rel="icon" href="./img/logo.png">
+    <meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
     <script src="./js/protobufjs.min.js"></script>
     <script src="./js/crypto-js.min.js"></script>
     <script src="./js/oauth-1.0a.min.js"></script>
     <script src="./js/jszip.min.js" defer></script>
-    <script src="./js/pdfkit.standalone.js" defer></script>
+    <script src="./js/jspdf.min.js" defer></script>
     <script src="./js/hls.light.min.js" defer></script>
     <script src="./js/ass.min.js" defer></script>
     <script src="./js/sql.min.js" defer></script>
-    <script src="./js/exif-js.min.js" defer></script>
```

Comment:
`exif` is required by one of the connectors to extract information from an image required for scraping and therefore should not be removed